src/components/CompareResults/ResultsFrameWorkDropDown.tsx

```diff
@@ -0,0 +1,62 @@
+import Box from '@mui/material/Box';
+import MenuItem from '@mui/material/MenuItem';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+import { frameworkMap } from '../../common/constants';
+import { Strings } from '../../resources/Strings';
+import { Spacing } from '../../styles';
+import type { Framework } from '../../types/types';
+
+const strings = Strings.components.searchDefault.sharedCollasped.framework;
+
+interface FrameworkDropdownProps {
+  frameworkId: Framework['id'];
+  onChange: (event: SelectChangeEvent) => void;
+}
+
+const sortFrameworks = (
+  frameworks: Record<Framework['id'], Framework['name']>,
+) => {
+  const unsortedArray = Object.entries(frameworks);
+
+  // Sort the array based on values
+  const sortedArray = unsortedArray.sort((a, b) => {
+    return a[1].localeCompare(b[1]);
+  });
+
+  return sortedArray;
+};
+
+const sortedFrameworks = sortFrameworks(frameworkMap);
+
+function ResultsFrameworkDropdown({
+  frameworkId,
+  onChange,
+}: FrameworkDropdownProps) {
+  return (
+    <Box display='flex' gap='1px' data-testid={'dropdown-select'}>
```

Comment:
I believe this `Box` isn't useful, you can remove it.
With an extra label, you don't need the test id either, because you'll be able to target the select with its label.